### PR TITLE
TASK: Skip symlink tests on windows when symlinking fails

### PR DESCRIPTION
**Pull request by albe**
On windows it is not unlikely that symlinking does not work due to missing access rights.
In that case a lot of the Files tests fail without any helpful notice and eventually lead
to the developer ignoring the failures, possibly overlooking real failures.

This change makes such symlink failures on Windows skip the test instead.

### CODE REVIEW

File: Neos.Utility.Files/Tests/Unit/FilesTest.php [hunk 1]
```diff
@@ -41,6 +41,25 @@ public function tearDown()
...
+     * @return boolean
+     * @throws \Exception
+     */
+    protected function trySymlink($target, $link)
+    {
+        try {
+            return symlink($target, $link);
+        } catch (\Exception $e) {
+            if (DIRECTORY_SEPARATOR !== '/') {
+                $this->markTestSkipped('Your windows installation does not allow symlinks. Make sure these tests are run with appropriate rights.');
```
Comment (bwaidelich) [inline Neos.Utility.Files/Tests/Unit/FilesTest.php]: What about "Your Windows Installation does not allow the PHP process to create symlinks. Make sure that tests are executed with appropriate file permissions"?

**Comment by albe** [comment]
> Did you actually test this on Windows with insuffizient permissions?

Yes, the description actually exactly describes my use-case. Those tests always failed for me (I never open an admin console to run the tests) and I started to just ignore those FilesTests, because I always assumed they were due to the missing rights for symlinks. Then I found there was a single actual failing test in between.

File: Neos.Utility.Files/Tests/Unit/FilesTest.php (hunk 1 above)
Comment (albe) [inline Neos.Utility.Files/Tests/Unit/FilesTest.php]: Maybe we could even be more concrete and state "Try running tests from an admin elevated command line".

**Comment by albe** [comment]
Changed the message according to comment from Bastian.

**Pull request merged**